google/resource_compute_backend_service.go

```diff
@@ -263,14 +270,19 @@ func resourceComputeBackendServiceDelete(d *schema.ResourceData, meta interface{
 	return nil
 }
 
-func expandBackends(configured []interface{}) []*compute.Backend {
+func expandBackends(configured []interface{}) ([]*compute.Backend, error) {
 	backends := make([]*compute.Backend, 0, len(configured))
 
 	for _, raw := range configured {
 		data := raw.(map[string]interface{})
 
+		g, ok := data["group"]
+		if !ok {
+			return nil, errors.New("google_compute_backend_service.backend.group must be set")
```

Comment:
This resource address may not actually be right, if the user names their backend something other than "backend". I'd prefer something along the lines of `fmt.Errorf("group field on backend %q must be set", backend.SelfLink)`, but that's just me.

Reply:
the address wouldn't contain the name of the backend though, that would be the value. the address would technically be something like `google_compute_backend_service.backend.<hash>.group` but I figured omitting the hash was fine in this case. We can't use the self link because there isn't a `compute.Backend` object to take the self link of

Reply:
I could use the index, I guess. Would that work better?

Reply:
Ah, no, I'm just confused. Ignore me, nitpick withdrawn. :)